### PR TITLE
Fix multiple graphQL endpoints

### PR DESCRIPTION
**Pull request by Gwillison415**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,20 +3,19 @@
...
-    "production-test": "scripts/production-test",
-    "dev": "scripts/dev",
-    "up": "scripts/up",
-    "down": "scripts/down",
-    "lint": "scripts/lint",
-    "production-lint": "scripts/production-lint",
-    "start": "scripts/production-start"
-  },
+	"scripts": {
+		"test": "scripts/test",
```
Comment (joshwyatt) [inline package.json:7]: @Gwillison415 I don't believe this indentation is what we are looking for.

File: package.json [hunk 2]
```diff
@@ -3,20 +3,19 @@
...
+		"up": "scripts/up",
+		"down": "scripts/down",
+		"lint": "scripts/lint",
+		"production-lint": "scripts/production-lint",
+		"start": "node index.js"
+	},
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@databraid/env-util": "1.1.2",
```
Comment (joshwyatt) [inline package.json:19]: @Gwillison415 I believe this library is required for production, could you tell me why you have removed it here?

File: src/models/Issue.js [hunk 3]
```diff
@@ -32,6 +32,12 @@ class Issue {
   get label() {
     return this.labels.map(label => new Label(label));
   }
+  get isPR() {
```
Comment (joshwyatt) [inline src/models/Issue.js:35]: @Gwillison415 I believe this work has been done in a previous PR.

**Comment by joshwyatt** [comment]
@Gwillison415 Thanks for the PR. You will need to pull down from upstream before we can finalize a review. I think after pulling down, some of the comments I left will have been resolved. Let me know after you have had a chance to pull down, address my comments, and push up again.

**Comment by Gwillison415** [comment]
this was related to the slack comment and quick fix to github api file but i think you handled the change before I got to it, and in addition - i didn't pull down first. the reason there is update to graphql schema and issue model is that micheal was working on my machine while my last logout PR was approved. closing now

**Pull request closed without merging**